src/Components/Header.js

```diff
@@ -48,8 +48,6 @@ class Header extends Component {
 
   handleClick = () => {
     this.auth.logout();
-    this.props.logout();
```

Comment:
Header feels like a dumb component to me, so I like that this was being passed in as a prop before.